_includes/download-available-cache-stores-band.html

```diff
@@ -1,8 +1,8 @@
 <div class="component-slim">
   <div class="grid-wrapper">
     <div class="width-12-12">
-      <h4>Available Cache Stores</h4>
-      <p>The following cache stores are part of the core Infinispan distribution and therefore are developed and released in-sync with Infinispan itself. Most of these cache stores are included in the <code>infinispan-embedded</code> JAR. Those which require additional dependencies are available in separate artifacts.</p>
+      <h4>Infinispan Cache Stores</h4>
+      <p>Infinispan offers several cache store implementations that are available with each release. Most cache stores are included in the <code>infinispan-embedded</code> JAR. Some others require additional dependencies and are available in separate artifacts.</p>
```

Comment:
`<code>infinispan-embedded</code> JAR` has been removed a while back

Reply:
nice one! I was going to ping you to ask so well spotted so quickly.

Reply:
hopefully the rewording sounds alright.